head/head.go

```diff
@@ -84,9 +91,7 @@ func NewHead(ctx context.Context, options ...opts.Option) (*Head, chan Bootstrap
 		return nil, nil, fmt.Errorf("failed to spawn libp2p node: %w", err)
 	}
 
-	dhtNode, err := dht.New(
-		ctx,
-		node,
+	o := []dht.Option{
```

Comment:
Developing the systems engineering traits of not using descriptive variable names, I see. Can I still bring you back? :)

Reply:
Let me fix that...